core/graphelier-service/db/mongo.go

```diff
@@ -19,11 +19,32 @@ type Datastore interface {
 	GetMessagesWithPagination(instrument string, paginator *models.Paginator) ([]*models.Message, error)
 	GetSingleMessage(instrument string, sodOffset int64) (*models.Message, error)
 	GetInstruments() ([]string, error)
+	RefreshCache() error
 }
 
 // Connector : A struct that represents the database
 type Connector struct {
 	*mongo.Client
+	cache cache
```

Comment:
When does this get populated initially?

Reply:
[Right here](https://github.com/Lercerss/graphelier/pull/129/files#diff-1717a8460c15d66b9eab79dbc9668be6R64) in the function `NewConnection`

Reply:
makes sense